SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/SpatialTestPossession/SpatialTestRemotePossession.h

```diff
@@ -15,10 +15,10 @@ struct FControllerPawnPair
 	GENERATED_BODY()
 
 	UPROPERTY()
-	ATestPossessionPlayerController* PlayerController;
```

Comment:
Is this required now? A drive-by or..?

Reply:
We now get spammed with a lot of "property is uninitialized" warnings/logs. There are actually about 20 more or something I still have to do. Would like to know what changed, would have to dig.

Reply:
(Maybe nulling of UProps is handled differently now? Only for certain primitive types? No idea)

Reply:
Maybe a bit pedantic but these nullptr fixes could maybe go in separate PR as it shouldn't be controversial and can just go into master, whereas the other stuff needs debate as you say.